basher.go

```diff
@@ -232,10 +233,21 @@ func (c *Context) Run(command string, args []string) (int, error) {
 	for _, arg := range args {
 		argstring = argstring + " '" + arg + "'"
 	}
+
+	sigChannel := make(chan os.Signal, 1)
+	signal.Notify(sigChannel)
+
 	cmd := exec.Command(c.BashPath, "-c", command+argstring)
 	cmd.Env = []string{"BASH_ENV=" + envfile}
 	cmd.Stdin = c.Stdin
 	cmd.Stdout = c.Stdout
 	cmd.Stderr = c.Stderr
-	return exitStatus(cmd.Run())
+
+	cmd.Start()
+
+	go func() {
+		sig := <- sigChannel
+		cmd.Process.Signal(sig)
```

Comment:
When the process exits, this will still be running. There is a potential edge case where it calls Signal when the process has exited. Might want to wrap the call to Signal with checking the `cmd.ProcessState` to see if it `Exited()`.